### DISCUSSION TITLE
Can the storagePrefix open to wallet connect core?

### DISCUSSION THREAD

**Discussion by lbhsot** [WalletConnect/walletconnect-monorepo#3469] [Developer Support]
let's say we have two tabs running same wallet connect page

1. **page A** wallet connect to **Protocol 1** and receive a transaction already
2. **page B** wallet connect to **Protocol 2**
3. **Protocol 1** send a transaction again, **page A** will not receive transaction anymore.

I guess the reason is wallet connect using the same `storagePrefix` `wc@2:core:0.3` so the connection in step 2 override step 1.

You can try using wallet connect in gnosis safe to reproduce this issue

**Comment by Comver** [comment]
Agree we have the same needs
